docs/src/main/docs/config/04_property-mapping.adoc

```diff
@@ -352,6 +375,21 @@ an instance of the POJO class initialized with the values passed as arguments.
 
 The following sections describe these patterns in more detail.
 
+This feature is available in Object mapping module, and is added through java service
+load mechanism. This is no longer part of core Config module, as it depends on reflection
```

Comment:
Did you mean the "Java ServiceLoader" class? If yes, please use that specific term.